src/client.toit

```diff
@@ -123,10 +143,30 @@ class Client:
 
   If neither is specified then the $default_port is used.
   */
-  get host/string --port/int?=null path/string --headers/Headers=Headers -> Response:
-    connection := new_connection_ host port --auto_close
-    request := connection.new_request GET path headers
-    return request.send
+  get host/string --port/int?=null path/string --headers/Headers=Headers --follow_redirect/bool=true -> Response:
```

Comment:
Should the parameter be called `follow_redirects` instead here and everywhere else?